Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: lint, pretty
 lint: 
-	flake8 --select=F --ignore= --ignore=F403,F405 --per-file-ignores=**__init__.py:F401 ./
+	flake8 --select=F --ignore= --ignore=F403,F405 --per-file-ignores=**__init__.py:F401 ./ --exclude=my_tests
```

Comment:
What are you excluding here, and why?

Reply:
Ahh, sorry, I shouldn't have committed the Makefile. It's just my local folder with tests that I store inside the repo. I'll change this back.